src/tile.js

```diff
@@ -154,11 +154,12 @@ $.Tile = function(level, x, y, bounds, exists, url, context2D) {
      */
     this.opacity    = null;
     /**
-     * The distance of this tile to the viewport center.
-     * @member {Number} distance
+     * The squared distance of this tile to the viewport center.
+     * Use for comparing tiles.
+     * @member {Number} squaredDistance
      * @memberof OpenSeadragon.Tile#
      */
-    this.distance   = null;
+    this.squaredDistance   = null;
```

Comment:
Re: breaking change - is there any public APIs for obtaining Tiles? If so, do any of them allow the user to obtain Tiles from the TiledImage? The latter would be the only case where the distance property has meaning, given that it is directly tied to display. I didn't find any while verifying Tile.distance was only used in TiledImage, but I could have missed something.

Sure, I'll make this private.